app.js

```diff
@@ -44,25 +47,74 @@ app.Views.TodoAppView = Backbone.View.extend({
         this.todoContent.append(this.subviews.todoListView.render());
 
         return this.$el;
+    },
+    createTodoOnEnter: function(e) {
+        const title = this.newTodoField.val().trim();
+        if (e.which == 13 && title != "") {
+
+            this.subviews.todoListView.collection.add(new app.Models.TodoItem({
+                title: title
+            }));
+        }
     }
 });
 app.Views.ToDoItemView = Backbone.View.extend({
     tagName: "li",
     className: "todo-item",
+    initialize: function() {
+        // this.listenTo(this.model, "destroy", this.remove);  // is this needed even though list view already listens for remove
+        this.listenTo(this.model, "change", this.render);
+    },
+    events: {
+        "click .js-delete-btn": "delete",
+        "dblclick .js-item-label": "startEditing",
+        "blur .js-edit-input": "finishEditing",        
+        "keypress .js-edit-input": "finishEditingOnEnter",
+        "change .js-complete-checkbox": "updateCompleted"
+    },
     template: Handlebars.compile($("#todo-item-template").html()),
     render: function() {
         console.log("rendering item view: ", this.model);
 
         this.$el.html(this.template(this.model.toJSON()))
+
+        this.editInput = this.$(".js-edit-input");
+        this.completeCheckbox = this.$(".js-complete-checkbox");
+
         return this.$el;
+    },
+    delete: function() {
```

Comment:
Why do you need a function which just wraps the basic backbone functionality? The consumer should be able to just call `model.destroy()` instead of a renamed wrapper function.

Reply:
delete is click event event `        "click .js-delete-btn": "delete",` 
Tried `        "click .js-delete-btn": "model.destroy",` but doesn't seem to work. 

Any other ways to approach this (pipe event directly to model.destroy) without introducing intermediary event handler?

Reply:
Ah I see why you are doing this now. Yea often the collection handles this. You could leave this, or you can have the model do `this.trigger('delete')`. The collection listens to this and will remove the model from the collection.

`destroy` does have a lot of things around server data management which you aren't using.

Could change or leave I guess